feature-libs/product-configurator/rulebased/components/group-menu/configurator-group-menu.component.ts

```diff
@@ -81,19 +87,14 @@ export class ConfiguratorGroupMenuComponent {
     protected configuratorGroupsService: ConfiguratorGroupsService,
     protected hamburgerMenuService: HamburgerMenuService,
     protected configRouterExtractorService: ConfiguratorRouterExtractorService,
-    protected configUtils: ConfiguratorStorefrontUtilsService
+    protected configUtils: ConfiguratorStorefrontUtilsService,
+    protected configGroupMenuService: ConfiguratorGroupMenuService,
+    protected directionService: DirectionService
   ) {}
+
   /**
-   * Prevents page down behaviour when users press space key to select buttons
+   * @deprecated since 4.0
```

Comment:
We need to introduce schematics for that and delete obsolete methods

Reply:
Done!